lib/plugins/aws/provider/awsProvider.test.js

```diff
@@ -17,6 +17,8 @@ const Serverless = require('../../../Serverless');
 const { replaceEnv } = require('../../../../tests/utils/misc');
 const { getTmpFilePath } = require('../../../../tests/utils/fs');
 
+const runServerless = require('../../../../tests/utils/run-serverless');
```

Comment:
Let's skip testing that. AJV is extensively tested on its own, we do not have to double confirm that `{ type: 'array' }` works, and sanity testing (that AJV is setup property) is already made in basic schema tests.